chantal/build.py

```diff
@@ -41,18 +41,23 @@ def build_job(args):
     if args.clone_location:
         job_state("running", "cloning repo")
 
+        if args.treeish:
+            branch = ("--branch %s " % args.treeish)
```

Comment:
we use treeish for exact commit hashes to check out, and --branch will only accept branches/tags, right?

Reply:
Yeah, I already changed this, and I'm testing these changes with the `depth` on project settings. Soon as possible, I'll update the draft, sorry for the slow implementation.

Reply:
haha no worries :) you're supposed to have fun, so take your time!